src/app/rating/rating.sample.ts

```diff
@@ -0,0 +1,18 @@
+import { Component } from '@angular/core';
+import { defineComponents, IgcRatingComponent } from 'igniteui-webcomponents';
+
+defineComponents(IgcRatingComponent);
+
+
+@Component({
+    selector: 'app-rating-sample',
+    styleUrls: ['rating.sample.css'],
+    templateUrl: 'rating.sample.html'
+})
+export class RatingSampleComponent {
+    public model = {
+        Name: 'BMW M3',
```

Comment:
Did you mean `Mercedes C 63 AMG`?

Reply:
I wanted to have a default score of around 5 so it would make more sense for a BMW

Reply:
Oh, I got it, the rating is on a 10-point scale :)